features.txt

```diff
@@ -7,235 +7,34 @@
 #
 # https://github.com/tc39/process-document
 
-# Missing checks in Proxy internal methods
-# https://github.com/tc39/ecma262/pull/666
```

Comment:
can we keep this PR link on the moved item?

Reply:
Sure — I found there was a bug in the linter that that triggered, but I've added a commit fixing it.

I assume you mean just the URLs to individual pull requests, that aren't easily discoverable given the name of the feature? I think the flags which correspond to whole proposals themselves are already pretty discoverable via tc39/proposals, so in the interest of avoiding clutter I didn't add those everywhere.